src/tests/functional/shared_test_classes/src/single_layer/prior_box.cpp

```diff
@@ -88,6 +90,8 @@ void PriorBoxLayerTest::SetUp() {
         shape_of_2,
         attributes);
 
+    ov::pass::disable_constant_folding(priorBox);
```

Comment:
This may affect to other plugins. Can we move it to any other GPU specific code?

Reply:
In essence, removing this from shared test class would mean that no plugin code will be executed in the test. I.e., it would give us a false confidence that some plugin works OK for this operation, while in fact plugin is not tested. Do we really need this?

Reply:
Understood your intension. If all build checks of Azure/Intel bot are pass, then it is ok  to locate here.